src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/.template.config/en/strings.json

```diff
@@ -4,6 +4,8 @@
     "name": "Blazor WebAssembly App",
     "description": "A project template for creating a Blazor app that runs on WebAssembly and is optionally hosted by an ASP.NET Core app. This template can be used for web apps with rich dynamic user interfaces (UIs).",
     "parameter.Hosted.name": "ASP.NET Core _hosted",
-    "parameter.PWA.name": "_Progressive Web Application"
```

Comment:
All the strings.json files can be removed, since in Dev17 they will never be loaded, since the template ships with templatestrings.en.json.

Instead you should add the following to localize/templatestrings.en.json
  "symbols/PWA/displayName": "Progressive Web Application",
  "symbols/Hosted/displayName": "ASP.NET Core Hosted",
  
  You may want to keep the '_' accelerators on the display names to preserve the keyboard shortcuts.

Reply:
> All the strings.json files can be removed, since in Dev17 they will never be loaded, since the template ships with templatestrings.en.json.

Will address that separately in a non patch change: https://github.com/dotnet/aspnetcore/issues/40107